plugins/source/aws/resources/services/autoscaling/groups.go

```diff
@@ -43,6 +43,16 @@ func Groups() *schema.Table {
 					PrimaryKey: true,
 				},
 			},
+			{
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: resolveCleanTags,
```

Comment:
why can't `client.ResolveTags` work on this? It just looks for a slice with `Key` and `Value` fields, but doesn't fail if the struct has any extras.

Reply:
That is cool! Didn't realize it could do that